### PR TITLE
TT-897: Dcs-client startup.sh should complete when it has deployed th…

### PR DESCRIPTION
**Pull request by MahmudH**
…e service

- The service runs in the background when service starts
- Tearsdown previous services upon startup
- There is more information of service upon startup, which includes service name and PID

Authors: @mahmudh @minhngocd

### CODE REVIEW

File: deploy/functions.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+function service_pid {
```
Comment (minhngocd) [inline deploy/functions.sh]: maybe let's rename this to get_service_pid?

File: deploy/functions.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  local service=$1
+  pids=`ps aux | grep java | grep ${service}.yml | awk '{print $2}'`
```
Comment (minhngocd) [inline deploy/functions.sh]: also can we rename pids to pid? since there's only 1 process

File: deploy/functions.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+function teardown_service {
+  service=$1
+  service_pid $service
+  for pid in $pids; do
```
Comment (minhngocd) [inline deploy/functions.sh]: since there's only 1 pid we can probably get rid of the for loop

File: startup.sh [hunk 4]
```diff
@@ -1,3 +1,24 @@
...
+service_name='dcs-client'
+
+. deploy/functions.sh
+
+teardown_apps
+
+echo -n "Starting $service_name..."
+
+./gradlew run > logs/dcs-client_console.log 2>&1 &
+
```
Comment (minhngocd) [inline startup.sh:14]: extra whitespace

**Comment by MahmudH** [comment]
Minno has reviewed this PR but does not have merge permissions so I will merge this PR.

**Pull request merged**